scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -1,7 +1,6 @@
 package scalafix.internal.patch
 
 import scala.annotation.tailrec
-import scala.collection.compat._
```

Comment:
https://github.com/scalacenter/scalafix/blob/d3fc7865cce73a4b38d5fe7171f1bc8ef1485ebb/project/ScalafixBuild.scala#L288 not handled in Scala 3